sorc/build_gdas.sh

```diff
@@ -19,12 +20,10 @@ while getopts ":j:dv" option; do
 done
 shift $((OPTIND-1))
 
-# TODO: GDASApp does not presently handle BUILD_TYPE
-
-BUILD_TYPE=${BUILD_TYPE:-"Release"} \
-BUILD_VERBOSE=${BUILD_VERBOSE:-"NO"} \
+# double quoting opts will not work since it is a string of options
+# shellcheck disable=SC2086
 BUILD_JOBS="${BUILD_JOBS:-8}" \
 WORKFLOW_BUILD="ON" \
-./gdas.cd/build.sh
+./gdas.cd/build.sh ${_opts}
```

Comment:
```suggestion
./gdas.cd/build.sh ${_opts} -f
```
Add `-f` to force building with a clean `build` directory to avoid stale cmake caches.